addons/website/views/website_templates.xml

```diff
@@ -965,21 +969,48 @@
     </div>
 </template>
 
+<template id="400">
+    <t t-call="website.layout">
+        <div id="wrap">            
+            <t t-if="request.debug">
+                <div class="container">
```

Comment:
You can factorize this `div` outside of the `t-if` `t-else` since you use it in both. (same in other views)